src/Batches/V4.js

```diff
@@ -6,23 +6,28 @@ export class V4 {
     this.api = api;
   }
 
-  /**
-   * @param {boolean} stopOnJobError
-   * @return boolean|number
-   */
-  create(stopOnJobError = false) {
-    return new Promise((resolve) => {
-      this.api.call('/v4/batch', { 'stop-on-job-error': Number(stopOnJobError) })
-        .then((response) => {
-          const batchId = response.success && response['batch-id'];
+    /**
+     * @param {boolean} stopOnJobError
+     * @param {string} callback
+     * @return boolean|number
+     */
+    create(stopOnJobError = false, callback = '') {
+        return new Promise((resolve) => {
+            let data = { 'stop-on-job-error': Number(stopOnJobError) };
+            if (callback.length > 4) {
```

Comment:
@alnarozhny why you are checking for the `callback` length? And why it should be greater than 4 to be a callback?